app/CocoaPods/CPPodfileReflection.m

```diff
@@ -105,6 +105,15 @@ - (void)parsePodfile;
   NSString *firstCharacter = [[description substringToIndex:1] uppercaseString];
   description = [firstCharacter stringByAppendingString:[description substringFromIndex:1]];
 
+  // Remove any raw instance description of a Podfile object
+  NSRegularExpression *regex = [NSRegularExpression regularExpressionWithPattern:@"#<Pod::Podfile:\\w+>"
```

Comment:
Could you make the regex slightly stricter by only matching `0x` and then hexadecimal characters instead of `\w+`? I’m just slightly paranoid :)